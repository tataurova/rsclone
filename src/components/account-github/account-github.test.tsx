import * as React from 'react';
import * as renderer from 'react-test-renderer';
import AccountGithub from './account-github';

describe('<AccountGithub />', () => {
  it('Render AccountGithub', () => {
    const tree = renderer
      .create(<AccountGithub
                address={'test@test.ru'}
                author={'Test Test'}
            />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
