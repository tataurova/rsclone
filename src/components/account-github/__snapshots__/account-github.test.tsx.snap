// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AccountGithub /> Render AccountGithub 1`] = `
<span
  className="makeStyles-link-1"
>
  <img
    alt="GitHub"
    className="makeStyles-img-2"
    src=""
  />
  <a
    className="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-linkColor-4 MuiTypography-colorPrimary"
    href="test@test.ru"
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
    rel="noreferrer"
    target="_blank"
  >
    Test Test
  </a>
</span>
`;
